DscResources/MSFT_EnvironmentResource/MSFT_EnvironmentResource.psm1

```diff
@@ -33,11 +43,12 @@ function Get-TargetResource
     [OutputType([Hashtable])]
     param
     (
-        [Parameter(Mandatory = $true)]
+        [Parameter(Mandatory = $true, HelpMessage="Indicates the name of the environment variable for which you want to ensure a specific state.")]
```

Comment:
Most other resources only have HelpMessages added only to Set, but these are added in Get?